### PR TITLE
ESM blog post engage page in French

### PR DESCRIPTION
**Pull request by Khutch25**
## Done

Updated the following /engage/fr/ pages to make them more consistent with other engage pages:
- /engage/fr/esm_1404
- /engage/fr/esm
- /engage/fr/openstack-made-easy

## QA

- Check out this feature branch
- Run `./run`
- Visit [/engage/fr/esm_1404](http://0.0.0.0:8001/engage/fr/esm_1404)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1T4Ar7n4Jwypecr96Wv5SQjdNdsISNkRHBU4DTxseHo0/edit#)
- Visit [/engage/fr/esm](http://0.0.0.0:8001/engage/fr/esm)
- See that the page matches the [copy doc](https://docs.google.com/document/d/14owLexmd5yn9g9Opq2oYAw-oxG2k2YKLgEi9bGqzzMg/edit)
- Visit [/engage/fr/openstack-made-easy](http://0.0.0.0:8001/engage/fr/openstack-made-easy)
- See that the page matches the [copy doc](https://docs.google.com/document/d/16u6gbZGqzIyJgjlOOks3DyW0fhIBgogV5yNAIzXOiDc/edit#heading=h.jzec7xymbc16)

### CODE REVIEW

**Comment by anasereijo** [comment]
Heyyyy @Khutch25 the takeover looks good 👍 but the landing page looks very different from what we usually do? (not sure if this based on a very old design?)

- There is no hero section defined - we usually have a strip with a background colour
- The icon is too big - make it 160px wide
- The intro text (just below the title) is too large - make it `h4`
- I think we usually have the form on the right side of the page? People will have to scroll a lot to actually see the form. Other solution could be having a button on the hero anchoring to the form.

Let me know if you need help!

Thanks

File: templates/engage/fr/esm_1404.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Le support d’Ubuntu 14.04 LTS est maintenant passé à ESM (Extended Maintenance Support){% endblock %}
```
Comment (solazio) [inline templates/engage/fr/esm_1404.html]: We should use the new metadata include https://github.com/canonical-web-and-design/www.ubuntu.com/pull/5255

File: templates/engage/fr/esm_1404.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    <p>Il est recommandé pour tous les utilisateurs de passer à la dernière LTS, Ubuntu 18.04. 18.04 est notamment plus rapide à booter, est basée sur le noyau 4.15, a été développé pour CI/CD avec Kubernetes, elle a les mitigations pour Spectre and Meltdown. Finalement, elle est aussi alignée avec le machine learning.</p>
+
+    <p>Il y a trois façon de faire une monté de verison -</p>
+
+    <ol>
+      <li>Utiliser l’interface graphique (Desktop seulement)</li>
+      <li>Entrer la commande suivante:</li> <br>
+
+      <pre class="wp-block-code">
+          <code>$ sudo do-release-upgrade</code>
```
Comment (solazio) [inline templates/engage/fr/esm_1404.html]: Please fix indentation

File: templates/engage/fr/esm_1404.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        </pre>
+
+      <li>Utilise la commande en utilisant Landscape, le cloud-management platform pour les machine Ubuntu.</li> <br>
+
+      <pre class="wp-block-code">
+          <code>$ sudo apt-get dist-upgrade</code>
+        </pre>
```
Comment (solazio) [inline templates/engage/fr/esm_1404.html]: Please fix indentation

**Comment by anasereijo** [comment]
thank you so much @sowasred2012! it's looking much better and consistent. 
One tiny thing on the /engage/fr/openstack-made-easy page. Can you please `Vertically center` the cloud icon?
Screenshot 2019-06-28 at 14 51 41

File: templates/engage/fr/esm_1404.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
+{% extends_with_args "engage/base_engage.html" title="Le support d’Ubuntu 14.04 LTS est maintenant passé à ESM (Extended Maintenance Support)" meta_description="Le support d’Ubuntu 14.04 LTS ‘Trusty Tahr’ est maintenant passé à ESM fin avril 2019, et ne sera plus supporté pour les utilisateurs sans ESM. ESM peut être contracté avec Ubuntu Advantage for Infrastructure." %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1T4Ar7n4Jwypecr96Wv5SQjdNdsISNkRHBU4DTxseHo0/edit{% endblock meta_copydoc %}
+
+
+{% block content %}
+<section class="p-strip--dark p-takeover--esm">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h1 class="p-takeover__title">Le support d’Ubuntu 14.04 LTS est maintenant passé à ESM (Extended Maintenance Support)</h1>
```
Comment (pmahnke) [inline templates/engage/fr/esm_1404.html]:
```suggestion
      <h1 class="p-takeover__title">Le support d’Ubuntu 14.04 LTS est maintenant passé à ESM</h1>
```

File: templates/engage/fr/esm_1404.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    <div class="col-4 u-hide--small u-align--center">
+      <img class="p-takeover__image" src="{{ ASSET_SERVER_URL }}9e59756d-shield-ESM.svg" alt="" class="u-hide--small">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip" lang="fr">
+  <div class="row">
+    <div class="col-8">
+      <p>Le support d’Ubuntu 14.04 LTS ‘Trusty Tahr’ est maintenant passé à ESM fin avril 2019, et ne sera plus supporté pour les utilisateurs sans ESM. ESM peut être contracté avec Ubuntu Advantage for Infrastructure.</p>
```
Comment (pmahnke) [inline templates/engage/fr/esm_1404.html]:
```suggestion
      <p>Le support d’Ubuntu 14.04 LTS ‘Trusty Tahr’ est maintenant passé à ESM (Extended Maintenance Support) fin avril 2019, et ne sera plus supporté pour les utilisateurs sans ESM. ESM peut être contracté avec Ubuntu Advantage for Infrastructure.</p>
```

**Comment by Khutch25** [comment]
LGTM! Thank you, @sowasred2012!

**Pull request merged**